CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 5.0.0-dev.1
+- Added support for generating ABI Specific integers.
+- Breaking: removed config keys - `size-map` and `typedef-map`.
```

Comment:
Should we have a deprecation period, so that people can migrate while still supporting the old keys but give warnings?

Reply:
I think it's not really necessary in light of the following
 - If the warnings are ignored these keys may override the default ABI specific integers (e.g for size_t) which will not be caught untill runtime on specific platforms.
- The size-map key will probably still need to be removed because of how it sort of clashes with the defaults.
- The lower SDK constraint probably needs to be updated to 2.16 by users.
- package:ffi will also now be mandatory for users as a dependency.
- The users will still be warned about having these old config keys, so it's not like this will go unnoticed.

So i guess a 5.0.0 makes more sense here(which allows us to add breaking changes).